geoportailv3/static/js/measure/measuredirective.js

```diff
@@ -31,7 +32,9 @@ app.measureDirective = function(appMeasureTemplateUrl) {
     restrict: 'E',
     scope: {
       'map': '=appMeasureMap',
-      'active': '=appMeasureActive'
+      'active': '=appMeasureActive',
+      'profiledata': '=appProfiledata',
```

Comment:
Rename `appProfiledata` to `appMeasureProfiledata` and `appProfileOpen` to `appMeasureProfileopen` for consistency.